sdk/search/azure-search-documents/src/samples/java/com/azure/search/documents/SearchClientJavaDocCodeSnippets.java

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.search.documents;
+
+import com.azure.core.credential.AzureKeyCredential;
+import com.azure.core.http.HttpPipeline;
+import com.azure.core.http.HttpPipelineBuilder;
+
+public class SearchClientJavaDocCodeSnippets {
```

Comment:
Should this be `SeachClientBuilderJavaDocCodeSnippets`?

Reply:
This is first part of the code snippet. I am planning to put SearchClient API code snippets in the same file.